crm-project/src/main/java/CRM/utils/Validations.java

```diff
@@ -176,23 +185,37 @@ public static Long validateToken(String token) {
     }
 
 
-    public static <T> void setContentToFieldIfFieldExists(T object, String fieldName, Object content) throws NoSuchFieldException {
+    public static <T> void setContentToFieldIfFieldExists(T object, UpdateField fieldName, Object content) throws NoSuchFieldException {
+        String fieldNameModified = fieldName.toString().replaceAll("_","");
         try {
             for (Field field : object.getClass().getDeclaredFields()) {
                 field.setAccessible(true);
                 Object value = field.get(object);
-                if (value == null) {
-                    continue;
-                }
-                if (!field.getName().equals(fieldName)) {
+                if (!field.getName().equalsIgnoreCase(fieldNameModified)) {
                     continue;
                 }
-                if (!(value.getClass().equals(content.getClass()))) {
+                if (value != null && !(value.getClass().equals(content.getClass()))) {
                     value.getClass().cast(content);
                 }
                 field.set(object, content);
                 return;
             }
```

Comment:
make this a function and put it as a private method inside Validations named "checkIfFieldExsistsInEntity"